### PR TITLE
Fix keys in HotkeyGroup

### PR DESCRIPTION
**Pull request by gettinToasty**

### CODE REVIEW

File: app/components/HotkeyGroup.vue [hunk 1]
```diff
@@ -14,7 +14,7 @@
     {{ title }}
   </h2>
   <div class="section-content" v-show="!collapsed">
-    <div v-for="hotkey in hotkeys" :key="hotkey">
+    <div v-for="hotkey in hotkeys" :key="`${hotkey.actionName}${hotkey.sceneId}${hotkey.sourceId}${hotkey.sceneItemId}`">
```
Comment (holiber) [inline app/components/HotkeyGroup.vue]: Hotkey class use `@ServiceHelper` decorator. So as an option, you can use `resourceId` as a unique id that this decorator appends to an instance.

Comment (gettinToasty) [reply app/components/HotkeyGroup.vue]: is there a reason `resourceId` isn't used in the `isSameHotkey` function in the hotkeys service?

Comment (holiber) [reply app/components/HotkeyGroup.vue]: `resourceId` was added when we exposed our API for the streamdeck team. The hotkeys code was added before

Comment (holiber) [reply app/components/HotkeyGroup.vue]: The current solution works fine. It's just FYI

**Pull request merged**